packages/eslint-plugin/package.json

```diff
@@ -50,6 +50,7 @@
     "@types/json-schema": "^7.0.3",
     "@types/marked": "^0.7.1",
     "@types/prettier": "^1.18.2",
+    "@types/react": "^16.9.2",
```

Comment:
do we really have to import react types? i don't see them being necessary to run this test